src/sentry/static/sentry/app/components/menuItem.tsx

```diff
@@ -58,18 +52,6 @@ class MenuItem extends React.Component<Props> {
         </Link>
       );
     }
-    if (this.props.href) {
-      return (
-        <a
-          title={this.props.title}
-          onClick={this.handleClick}
-          href={this.props.href}
-          tabIndex={-1}
-        >
-          {this.props.children}
-        </a>
-      );
-    }
```

Comment:
Are we sure this isn't in use? Can we be a bit more defensive restore this code and do something similar in `Link` where we track to Sentry if `this.props.href` exists. My fear is that this is still in use somewhere and we do not have any visibility if it does since it will not cause any errors but it will still appear broken to the user.